doc/contents/CodeCellExample.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import jupyter.Displayer;\n",
+    "import jupyter.Displayers;\n",
+    "import com.twosigma.beakerx.CodeCell;\n",
+    "\n",
+    "Displayers.register(CodeCell.class, new Displayer<CodeCell>() {\n",
```

Comment:
put the displayer into the regular CodeCell class so it's always available to the user.

the displayer should show all the fields, not just the source.